intermediate_html_css/intermediate_css_concepts/advanced_selectors.md

```diff
@@ -52,12 +52,12 @@ main div {
 But what if we wanted to be more specific and <span id="childvdesc-knowledge-check">select *only* the `child` or `grand-child` divs?</span> That's where the child combinator `>` comes in handy. Unlike the descendant combinator, it will only select direct children.
 
 ```css
-/* This rule will only select divs with a class of child */
+/* This rule will only select divs with the class child */
```

Comment:
This would make it more consistent with the other code block, and also more directly refer to what divs are selected from the above HTML, as opposed to that rule generally selecting divs with those classes (which it won't).
```suggestion
/* This rule will only select the divs with the class child */
```